layout-server/src/main/resources/LayoutServer.run.xml

```diff
@@ -0,0 +1,13 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="LayoutServer" type="SpringBootApplicationConfigurationType"
+    factoryName="Spring Boot">
+    <option name="ALTERNATIVE_JRE_PATH" value="BUNDLED"/>
+    <option name="ALTERNATIVE_JRE_PATH_ENABLED" value="true"/>
+    <module name="layout-server"/>
+    <option name="SPRING_BOOT_MAIN_CLASS"
+      value="org.finos.vuu.layoutserver.LayoutServerApplication"/>
+    <method v="2">
+      <option name="Make" enabled="true"/>
+    </method>
+  </configuration>
+</component>
```

Comment:
This config isn't working for me. I get the option to run it, but when I try to do so, I get the error below. I think this might be a consequence of using IntelliJ Community Edition - do you know if there are any ways to set run configs?
![image](https://github.com/ScottLogic/finos-vuu/assets/79100986/46e282d5-c6e9-4e73-a427-1ac0213dd774)

Reply:
Replaced with yours [04ae976](https://github.com/ScottLogic/finos-vuu/pull/98/commits/04ae976e2636c241459a3d491539150cfe1e0d6e). It's quite hard to test this works, my IDE seems to overwrite it whenever I run it. Maybe @vferraro-scottlogic can triple check?

Reply:
works fine for me